.github/workflows/build_and_test.yml

```diff
@@ -248,3 +248,17 @@ jobs:
         cmake .. -DBUILD_JAVA=OFF -DPROTOBUF_HOME=${CMAKE_PREFIX_PATH}
         make package test-out
 
+  meson:
+    name: "Meson C++ configuration"
+    runs-on: ubuntu-latest
```

Comment:
1. To make it sure the supported OSes, please use OS matrix like the existing system at least.

https://github.com/apache/orc/blob/63541a231b9c7317c755b675dd3ec10541735c0d/.github/workflows/build_and_test.yml#L62-L72

2. In addition to the above OSes, CI has the following test coverage too. However, I believe (1) is enough for `Meson`.

https://github.com/apache/orc/blob/63541a231b9c7317c755b675dd3ec10541735c0d/.github/workflows/build_and_test.yml#L40-L51